assets/js/userCtrl.js

```diff
@@ -493,16 +493,16 @@ define(['lodash', 'regex'], function (_, regex) {
 
       if (applicationFlair.sub === "pokemontrades" &&
         currentFlair &&
-        currentFlair.trades > trades &&
-        currentFlair.involvement > involvement &&
-        currentFlair.giveaways > giveaways) {
+        currentFlair.trades >= trades &&
+        currentFlair.involvement >= involvement &&
+        currentFlair.giveaways >= giveaways) {
         return false;
       }
 
       if (applicationFlair.sub === "svexchange" &&
         currentFlair &&
-        currentFlair.eggs > eggs &&
-        currentFlair.giveaways > giveaways) {
+        currentFlair.eggs >= eggs &&
+        currentFlair.giveaways >= giveaways) {
         return false;
       }
 
```

Comment:
Example:

* Poke Ball flair requirements: 10 trades, 0 giveaways
* Premier ball flair requirements: 20 trades, 0 giveaways

Before, a user with Premier Ball flair would be allowed to apply for Poke Ball flair, because Premier Ball requires no more giveaways than Poke Ball flair. This is fixed now.